rest/client.go

```diff
@@ -92,11 +96,9 @@ func (c *client) DeleteComponent(ctx context.Context, id uuid.UUID) error {
 		Assign("component", id).
 		SetHeader("Accept", "application/json")
 
-	if _, err := c.Do(ctx, request); err != nil {
-		return err
-	}
+	_, err := c.Do(ctx, request)
```

Comment:
Isn't it cleaner to keep the return err on a separate line from return nil? If only that stack traces are easier to debug?

Reply:
it's more code preference, I can revert it

Reply:
fixed